bubuku/cli.py

```diff
@@ -92,37 +100,34 @@ def trigger_preferred_replica_election(dry_run: bool, max_json_size: int):
                 _LOG.info("Found incorrect assignment: %s, leader is %d, but should be the first one in %s",
                           key, leader, replica_list)
                 wrong_assignment.append(key)
-        while wrong_assignment:
-            items_to_take = len(wrong_assignment)
-            change_applied = False
-            while not change_applied:
-                json_element = {
-                    "version": 1,
-                    "partitions": [{'topic': v[0], 'partition': int(v[1])} for v in wrong_assignment[:items_to_take]]
-                }
-
-                res = json.dumps(json_element).encode('utf-8')
-                if len(res) > max_json_size:
-                    new_items_to_take = int(items_to_take / 2)
-                    _LOG.info("Not fitting to %d bytes with %d items, will try %d items",
-                              max_json_size, items_to_take, new_items_to_take)
-                    items_to_take = new_items_to_take
-                    if items_to_take <= 0:
-                        _LOG.error("Incorrect configuration - even one key is not fitting to proposed size %d. "
-                                   "Stop playing and do the job!", max_json_size)
-                        exit(1)
-                    continue
-                if dry_run:
-                    print(res.decode('utf-8'))
-                else:
+
+        if dry_run:
```

Comment:
is that what you added ?

Reply:
What do you mean?

Reply:
there is a big removal and addition of the code. I asked if you added only if clause with `_dump_replica_assignment_as_json` or you also changed something in removed/added code

Reply:
I see.  It's easier to see if you hide "whitespace-only" changes: https://github.com/zalando-nakadi/bubuku/pull/178/files?w=1

Reply:
I did not intend to change the logic, it's just different nesting.